onnxruntime/__init__.py

```diff
@@ -12,7 +12,12 @@
 as Deep Learning algorithms in the
 `ONNX-ML format <https://github.com/onnx/onnx/blob/master/docs/IR.md>`_.
 """
-__version__ = "0.3.0"
+import os
+
+__version__ = ''
+cwd = os.path.dirname(os.path.realpath(__file__))
+with open(cwd + '/../VERSION_NUMBER') as f:
```

Comment:
Please dont hardcode the path.. use path.join instead

Reply:
Good point